src/main/java/repository/PersonDaoImpl.java

```diff
@@ -0,0 +1,40 @@
+package repository;
+
+import lombok.extern.log4j.Log4j;
+import model.Person;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Log4j
+@Repository
+@Qualifier("personDao")
+public class PersonDaoImpl implements PersonDao {
+    @Autowired
+    JdbcTemplate jdbcTemplate;
```

Comment:
add access modificator
and read about constructor Injection in Spring with lombok https://www.baeldung.com/spring-injection-lombok
and why field injection is not recommended http://blog.marcnuri.com/field-injection-is-not-recommended/